src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -194,23 +193,28 @@ internal override bool IsCallerMemberName
             }
         }
 
-        private ConstantValue DefaultSyntaxValue
+        private ConstantValue GetDefaultSyntaxValue(DiagnosticBag diagnosticsOpt = null)
```

Comment:
Somewhat, I think we should explicitly track API changes what we would like to revert. We can keep the list in that issue.